### PR TITLE
dev-prod

### PR DESCRIPTION
**Pull request by tehpsalmist**

### CODE REVIEW

**Comment by tehpsalmist** [comment]
hey, if we approve this, does that mean we'll publish again? If so, can I do it? I want to make sure that publishing is also good on "anyone's" machine.

File: packages/zengine-ui/.gitignore [hunk 1]
```diff
@@ -1 +1 @@
-lib
+style.*
```
Comment (alexweber) [inline packages/zengine-ui/.gitignore]: @tehpsalmist maybe add forward slash here to make path explicit, we dont want to ignore all styles necesasrly just top level build artifacst

File: packages/zengine-ui/package.json [hunk 2]
```diff
@@ -7,16 +7,18 @@
   "scripts": {
     "start": "npm run sass -- --watch",
     "watch": "npm run sass -- --watch",
-    "sass": "sass scss/style.scss lib/style.css",
-    "sass:min": "sass scss/style.scss lib/style.min.css --style compressed",
-    "build": "rimraf lib; npm run sass; npm run sass:min; cpy scss lib --parents; cpy package.json README.md CHANGELOG.md lib",
```
Comment (alexweber) [inline packages/zengine-ui/package.json]: @tehpsalmist we can remve rimraf and cpy from dev deps

Comment (tehpsalmist) [reply packages/zengine-ui/package.json]: I thought I did

File: packages/zengine-ui/.gitignore (hunk 1 above)
Comment (tehpsalmist) [reply packages/zengine-ui/.gitignore]: it turns out, by omitting the slashes, this path actually means `./style.*`

Comment (tehpsalmist) [reply packages/zengine-ui/.gitignore]: so we're good.

File: packages/zengine-ui/package.json (hunk 2 above)
Comment (alexweber) [reply packages/zengine-ui/package.json]: maybe i missed the diff no worries thx

**Pull request merged**